learn/features/staking.md

```diff
@@ -71,28 +71,36 @@ If you're looking for candidate or collator-specific requirements and informatio
 
 === "Moonbeam"
-    |      Variable       |                                                  Value                                                  |
-    |:-------------------:|:-------------------------------------------------------------------------------------------------------:|
-    | List of candidates  |                [Moonbeam Subscan](https://moonbeam.subscan.io/validator){target=_blank}                 |
-    | Collator statistics | [Moonbeam Explorer](https://moonbeam-explorer.netlify.app/stats/miners?network=Moonbeam){target=_blank} |
+    |           Variable           |                               Value                               |
+    |:----------------------------:|:-----------------------------------------------------------------:|
+    |     Stake GLMR Dashboard     |        [Stake GLMR](https://stakeglmr.com/){target=_blank}        |
+    |    Collators Leaderboard     | [Moonscan](https://moonbeam.moonscan.io/collators){target=_blank} |
+    | Staking Simulation Dashboard |      [Web3Go](https://web3go.xyz/#/Moonbeam){target=_blank}       |
 
 === "Moonriver"
-    |      Variable       |                                                                 Value                                                                 |
-    |:-------------------:|:-------------------------------------------------------------------------------------------------------------------------------------:|
-    | List of candidates  |                              [Moonriver Subscan](https://moonriver.subscan.io/validator){target=_blank}                               |
-    | Collator statistics |               [Moonbeam Explorer](https://moonbeam-explorer.netlify.app/stats/miners?network=Moonriver){target=_blank}                |
-    |  Collator APY data  | [DappLooker Collator Dashboard](http://analytics.dapplooker.com/public/dashboard/7dfc5a6e-da33-4d54-94bf-0dfa5e6843cb){target=_blank} |
+    |           Variable           |                                                                 Value                                                                 |
+    |:----------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------:|
+    |     Stake MOVR Dashboard     |                                          [Stake MOVR](https://stakemovr.com/){target=_blank}                                          |
+    |    Collators Leaderboard     |                           [Moonscan for Moonriver](https://moonriver.moonscan.io/collators){target=_blank}                            |
+    | Staking Simulation Dashboard |                                        [Web3Go](https://web3go.xyz/#/Moonriver){target=_blank}                                        |
+    |      Collator APY data       | [DappLooker Collator Dashboard](http://analytics.dapplooker.com/public/dashboard/7dfc5a6e-da33-4d54-94bf-0dfa5e6843cb){target=_blank} |
```

Comment:
Correct Dapplooker dashboard: https://network.dapplooker.com/moonriver/collator 

I would put Dapplooker Dashboard (instead of APY data) we can't mention APY